apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.ts

```diff
@@ -55,7 +56,7 @@ router.get(
     const editor = new QuestionTransferEditor({
       locals: res.locals,
       from_qid: qid,
-      from_course_short_name: file_transfer.from_course.short_name,
+      from_course_short_name: file_transfer.from_course.short_name ?? '',
```

Comment:
Nit: consider pushing the null handling down into `QuestionTransferEditor`. On the off chance that this is nullish, this would result in a commit message containing `(from course )`. It's probably better to say `(from unknown course)` or something similar instead.

Reply:
Done in f7f56db0079eea6216b0c0375daca4ce5a79e78f.